src/bun.js/test/jest.classes.ts

```diff
@@ -146,11 +146,9 @@ export default [
       },
       toHaveBeenCalledWith: {
         fn: "toHaveBeenCalledWith",
-        length: 1,
       },
       toHaveBeenLastCalledWith: {
         fn: "toHaveBeenLastCalledWith",
-        length: 1,
       },
```

Comment:
Since they can have an unlimited number of arguments, it didn't seem like the length property would apply here